reference/orders.v2.oas2.yml

```diff
@@ -96,6 +96,8 @@ paths:
         To update a product in an order, include `id` in the body. The body should only contain the fields that need to be updated. Those fields that are omitted will not be changed.
         
         To remove a product from an order, set that product’s `quantity` to `0`.
+
+        All discounts and promotions applied to the order are cleared after the update. Since the order data syncs with other ERP systems, like Amazon or eBay, the updated order returns to the default setting removing any applied discounts.
```

Comment:
```suggestion
After the update, the PUT request clears all discounts and promotions applied to the order. Since the order data syncs with other ERP systems, like Amazon or eBay, the updated order returns to the default setting, removing any applied discounts.
```